externs/browser/w3c_css.js

```diff
@@ -2053,6 +2053,20 @@ CSSProperties.prototype.userSelect;
 
 // http://www.w3.org/TR/cssom-view/#extensions-to-the-window-interface
 
+// CSS 3 Images
+
+/**
+ * @type {string}
+ * @see https://drafts.csswg.org/css-images-3/#the-object-fit
```

Comment:
I think you want to use https://www.w3.org/TR/css3-images/#object-fit?
The drafts.csswg.org seems suspicious. 
I'm not very familiar with Candidate Recommendation vs Editor's Draft. Do you have more context on that?

Reply:
Yes indeed!

I'm afraid there is no context, I have blindly followed the link from the MDN page.